### PR TITLE
[ADD] sales: point of sale qr code payment

### PR DESCRIPTION
**Pull request by tong-odoo**
This PR adds a new doc detailing how to setup point of sale qr code payment.

### CODE REVIEW

**Comment by LoredanaLrpz** [comment]
Hey @tong-odoo , thank you for your work on this doc.
I'll be sending my review asap

**Review by LoredanaLrpz — APPROVED** [review]

File: content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+   :header-rows: 1
+
+   * - QR code types
+     - Module name
+     - Technical name
+     - Description
+   * - Pix
+     - :doc:`Brazilian - Accounting<../../../finance/fiscal_localizations/brazil>`
+     - `l10n_br`
+     - The base module to manage chart of accounting and localization for Brazil.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst:33]:
```suggestion
     - The base module to manage the chart of accounts and localization for Brazil.
```
To be corrected in the other descriptions as well.

File: content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+     - :doc:`Account SEPA QR Code<../../../finance/accounting/customer_invoices/epc_qr_code>`
+     - `account_qr_code_sepa`
+     - This module adds support for SEPA Credit Transfer QR-code generation.
+
+Create the payment method
+-------------------------
+
+#. Open the Point of Sale application.
+#. Go to :menuselection:`Configuration --> Payment Methods` and create a payment method.
+#. Set a bank typed journal.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst:68]:
```suggestion
#. Set a bank-type journal.
```

File: content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+   - Select :guilabel:`SEPA Credit Transfer QR` if you are part of the Single Euro Payments Area
+     (SEPA).
+   - Select :guilabel:`EMV Merchant-Presented QR-code` for other QR code types.
+
+.. image:: qr_code_payment/qr-payment-methods-setting.png
+   :alt: QR code payment method configuration
+
+.. important::
+   At least one bank account must be defined on the journal to allow registering QR code payments
+   with Bank apps.
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst:81]:
```suggestion
   with bank apps.
```

File: content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+Once the payment method is created, you can select it in your POS settings. To do so, go to the
+:ref:`POS' settings <configuration/settings>` and add the payment method under the
```
Comment (xpl-odoo) [inline content/applications/sales/point_of_sale/payment_methods/qr_code_payment.rst:84]:
```suggestion
Once the payment method is created, go to the
:ref:`POS' settings <configuration/settings>` and add the payment method under the
```

**Comment by LoredanaLrpz** [comment]
@robodoo r+

**Pull request closed without merging**